app/assets/stylesheets/application.scss

```diff
@@ -609,9 +611,14 @@ label {
 
 .label {
   width: auto;
-  display: inherit;
+  display: initial;
```

Comment:
This change broke the label in sign in
<img width="1077" alt="screen shot 2016-11-19 at 8 15 31 pm" src="https://cloud.githubusercontent.com/assets/3010728/20460377/f48d8c16-ae94-11e6-9f82-5670bf362dc5.png">

(The labels should be blocks)